docs/reference/changelog-r2021.md

```diff
@@ -5,6 +5,7 @@ Released on XX, XXth, 2021.
 
   - Enhancements
     - Improved performance on [Lidar](lidar.md) point cloud generation ([#3499](https://github.com/cyberbotics/webots/pull/3499)).
+    - Improved UX when using the object rotation around axis feature ([#3540](https://github.com/cyberbotics/webots/pull/3540)).
```

Comment:
I'm not sure that users that will read the change log will understand what is UX.